src/test/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/EventSpec.scala

```diff
@@ -2903,6 +2905,152 @@ class EventSpec extends Specification {
       val eventJson = event.toJson(false)
       eventJson.as[Event] must beRight(event)
     }
+
+    "successfully decode object with event which has no contexts or unstruct_event" in {
```

Comment:
It's not very clear how this test relates to the decoding bug the commit is fixing.

Reply:
The bug was relted to any class with nested `Event` (or more specifically `Contexts` or `UnstructEvent`). Here, `Test` is such class.